common/src/main/java/org/wildfly/httpclient/common/Version.java

```diff
@@ -0,0 +1,24 @@
+package org.wildfly.httpclient.common;
+
+/*
+ * Versioning enum for HttpHandler implementations.
+ */
+public enum Version {
```

Comment:
The current protocol versions class can be edited to incorporate this... I imagine you are probably thinking on doing the same already

Reply:
Hi @fl4via sorry, but which protocol versions class are you referring to? 
Do you mean that for each wire protocol version you will have a set of handlers versioned for that wire protocol? e.e. EJB client wire protocol v2 will have a set of v2 HTTP client handlers; EJB client wire protocol 3 will have a set of v3 HTTP handlers?

Reply:
I meant the Protocol class @rachmatowicz , to me it makes sense to keep everything related to protocol in a single class.

Reply:
Do you mean org.jboss.ejb.protocol.remote.Protocol? I can change that once I get things working.

Reply:
@rachmatowicz yes, so I suggest renaming Protocol to ProtocolVersion or something like that and transforming htat into an enum. I think it makes sense having everything at a single place,since they contain related stuff.